Static/DataSource.swift

```diff
@@ -169,7 +169,7 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return sectionForIndex(section)?.header?.viewHeight ?? 0
+        return sectionForIndex(section)?.header?.viewHeight ?? UITableViewAutomaticDimension
```

Comment:
For some reason, having `0` here prevents the first section title from appearing. Weird.